ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/KsqlVersionCheckerAgent.java

```diff
@@ -29,6 +29,8 @@
 
   private boolean enableSettlingTime;
 
+  private final ActiveChecker activeChecker = new KsqlServerActiveCheckerImpl();
```

Comment:
No need for this field / type with my suggested changes.

```suggestion
  private volatile long lastRequestTimestamp;
```